scripts/wawCommons.py

```diff
@@ -526,4 +526,85 @@ def replaceValue(sourceJson, target, replacementJson, matchKey = True):
             replacedValuesNumber += 1
     return targetJson, replacedValuesNumber
 
+def getFunctionResponseJson(CFNamespaceUrl, username, password, package, functionName, parameters, data):
```

Comment:
I would like to change `CFNamespaceUrl` to something that starts with lower letter... we could probably also consider to split it to endpoint and namespace.

Reply:
Splitted.